meilisearch/index.py

```diff
@@ -1917,6 +1918,68 @@ def reset_search_cutoff_ms(self) -> TaskInfo:
 
         return TaskInfo(**task)
 
+    # PROXIMITY PRECISION SETTINGS
+
+    def get_proximity_precision(self) -> ProximityPrecision | None:
```

Comment:
I don't think this will ever be `None`. If it hasn't been set it should be the default.

```suggestion
    def get_proximity_precision(self) -> ProximityPrecision:
```